src/test/resources/dev.application.properties

```diff
@@ -0,0 +1,11 @@
+#Jdbc
+jdbc.url=jdbc:postgresql://localhost:5432/test-agregator-service
+jdbc.user=postgres
+jdbc.password=root
+appPort=8080
+
+#Flyway
+flyway.url=jdbc:postgresql://localhost:5432/test-agregator-service
+flyway.user=postgres
+flyway.password=root
+flyway.driver=org.postgresql.Driver
```

Comment:
This file should be add to .gitignore and remove from index of commit
Add to .gitignore
Add gitignore file to index of commit
Remove dev.application.properties from index of coomit:
 git rm --cached src/test/resources/dev.application.properties